rllib/agents/a3c/README.md

```diff
@@ -0,0 +1,22 @@
+# Advantage Actor-Critic (A2C, A3C)
+
+## Overview 
+
+[Advantage Actor-Critic](https://arxiv.org/pdf/1602.01783.pdf) proposes two distributed model-free off-policy RL algorithms, A3C and A2C. These algorithms are distributed versions of DQN with different distributed execution patterns. The paper accelerates training via scaling data collection, i.e. introducing worker nodes, which carry copies of the central node's Q-network, that collect data from the environment in parallel. 
```

Comment:
A3C and A2C are both on-policy and neither have a Q-network (you probably mean "versions of PG" and "central node's policy network").